client/python/armada_client/client.py

```diff
@@ -146,29 +146,67 @@ def reprioritize_jobs(
         response = self.submit_stub.ReprioritizeJobs(request)
         return response
 
-    def create_queue(self, name: str, **queue_params) -> empty_pb2.Empty:
+    def create_queue(
+        self,
+        name: str,
+        priority_factor: Optional[int],
+        user_owners: Optional[List[str]] = None,
+        group_owners: Optional[List[str]] = None,
+        resource_limits: Optional[Dict[str, float]] = None,
+        permissions: Optional[List[submit_pb2.Queue.Permissions]] = None,
```

Comment:
Should we go a little farther with Permissions?  Have this also take pythonic types as opposed to the proto representations?

Reply:
Would we want this done in a seperate PR, and leave this as is for now? Or do that in this one?

Reply:
I'd put it in this one.

Reply:
Cool! Will work on this shortly :))